License2Deploy/rolling_deploy.py

```diff
@@ -11,6 +11,7 @@
 class RollingDeploy(object):
 
   MAX_RETRIES = 10
+  INSTANCE_NAME = "{0}-{1}{2}-{3}" #e.g. aw1-dnbi-cirrus107-qa
```

Comment:
are you missing a hyphen between {1} and {2}? Or is the hyphen part of one of those?